components/navbar.tsx

```diff
@@ -86,11 +101,13 @@ export function Navbar() {
         icon={<RiMenu4Fill />}
       />
 
-      <Link href="/">
-        <a>
-          <Image width={50} height={50} src={Logo} alt="logo" />
-        </a>
-      </Link>
+      <Box display={{ base: "none", md: "flex" }}>
+        <Link href="/">
+          <a>
```

Comment:
Why `a` tag inside Link?

Reply:
Thanks for pointing it out. It was a glitch.